feg/gateway/fabfile.py

```diff
@@ -18,48 +18,63 @@
 sys.path.append('../../orc8r')
 import tools.fab.dev_utils as dev_utils  # NOQA
 import tools.fab.types as types
-from fabric.api import local
+import tools.fab.vagrant as vagrant
+from fabric import Connection, task
 
 SNOWFLAKE_FEG_FILE = '../../.cache/feg/snowflake'
 NETWORK_ID = 'feg_test'
 FEG_DOCKER_LOCATION = 'docker/'
+FEG_INTEG_DOCKER = "$MAGMA_ROOT/lte/gateway/python/integ_tests/federated_tests/docker"
+AGW_ROOT = '../../lte/gateway'
 
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
 
-def register_feg_gw(location_docker_compose: str = FEG_DOCKER_LOCATION):
+@task
+def register_feg_gw(c, location_docker_compose=FEG_DOCKER_LOCATION):
     """
     Add FEG gateway to orc8r
     Args:
+        c: fabric Connection
```

Comment:
Does a linter complain if `c` is not documented? Genuinely unsure if we want to have this:
```
~/magma/feg/gateway$ fab --help register-feg-gw
```
also prints the complete docstring, i.e., could be confusing to not use it in a command line call. What do you think?